src/components/footer/icons.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Box, Link, useTheme } from '@chakra-ui/core';
+import footerStyles from './Footer.module.css';
+
+/**
+ * Icon container with external link
+ * @param {string} The external link the icon points to
+ */
+const IconContainer = ({ children, link }) => {
```

Comment:
Is there any chance we can separate these icons out into their own individual files within `src/components/SVG/` - this will ensure these icons can be leveraged in different components.

Reply:
Sure